db/upgrade.php

```diff
@@ -182,6 +115,73 @@ function xmldb_tincanlaunch_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2015033100, 'tincanlaunch');
     }
 
+    if ($oldversion < 2015112702) { //New version in version.php
```

Comment:
Just saw a commit by Avi Levi 
https://github.com/garemoko/moodle-mod_tincanlaunch/commit/5a9590f7f24825dca657976fd93333326c2a0229
(that works on my team, and which is currently upgrading all of the modules we use to Moodle 3.0), changed a version on the version.php file. Is that might be related?